app/models/google_event.rb

```diff
@@ -29,6 +39,42 @@ def list_events(credentials, starting, ending)
       events
     end
 
+    def format_params(params)
+      params.merge(
+          start: {date_time: DateTime.parse(params[:start_time]).rfc3339(9)},
+          end: {date_time: DateTime.parse(params[:end_time]).rfc3339(9)}
+      ).except(:start_time, :end_time, :conference_room_id, :permitted).to_h
+    end
+
+    def create(credentials, conference_room_ids, event_data = {})
+      event_data = event_data.try :deep_symbolize_keys
```

Comment:
Do you need `try`? Can `event_data` be nil? You're shadowing out the parameter because both a variable and a parameter are called `event_data`